code/ui/golden/BIP32Path.json

```
@@ -0,0 +1,10 @@
+{
+    "samples": [
+        "",
+        "1911087457/2025199967H",
+        "",
+        "9886650H/131789259H",
+        "1324835599H"
+    ],
+    "seed": 300465375
+}
```

```suggestion
}

```

Newline at end of file?